### DISCUSSION TITLE
How to configure the "Request a change to this listing" form?

### DISCUSSION THREAD

**Discussion by toddjsaunders** [code4recovery/12-step-meeting-list#658] [Q&A] [answered]
Are there instructions somewhere on how to configure the "Request a change to this listing" form on the meeting detail page?

I've searched and couldn't find any information.

Thanks!

**Comment by kiyote33** [comment]
Hi @toddjsaunders, the only configuration or setup necessary to activate the "Request a change to this listing" feature is to enter a valid email address entered in the **User Feedback Emails** admin box on the Meetings/Import & Settings page. That will allow display of a simple feedback form to be filled out and submitted to an admin by an end user.  Please advise if this isn't the information you are looking for... If your having a display problem (i.e... the submit button is not showing) try adding Additional CSS: **#tsml #feedback .list-group-item-form {	font-size: 80%; }** to reduce the font size...

**Reply by toddjsaunders** [reply]
I've had an email address there since I've starting working with TSML, it populated my admin email automatically. I've tried removing that and adding a different email, no change.

Our form looks like this:
![Screen Shot 2022-01-31 at 3 23 43 PM](https://user-images.githubusercontent.com/8978087/151889650-b10875ad-a400-48e5-bf31-bd93ac135ba0.png)

It's missing the Your Name, Email address and Message fields..

Thanks!

**Reply by kiyote33** [reply]
@toddjsaunders - if you can tell me the theme you are using I may be able to provide you with the CSS to get your form to display correctly. I'm guessing you will need something like **#tsml #feedback .list-group-item-form { width: 100%; height: 100%; }**.  If I can duplicate your issue in my sandbox, then I can have a higher degree of certainty in the answer provided.

**Reply by toddjsaunders** [reply]
Hi,

Our theme is Customizable Blogily by ThemeEverest.

Thanks!

**Comment by kiyote33** [comment, marked as answer]
Hi @toddjsaunders 
Thanks for providing the theme information. I was able to replicate your issue after installing your theme, which for some reason messed with our CSS.  I was able to get around it by inserting some override CSS which you can see in the following image:
![image](https://user-images.githubusercontent.com/57125427/152212735-a03b5889-4cc1-471b-a60e-99a5468774c8.png)
Hope this works for you. Let me know if we can be of any more help to you...

**Reply by toddjsaunders** [reply]
That worked like a charm -- thanks so much!
